cookbooks/bcpc/recipes/zabbix-head.rb

```diff
@@ -225,3 +231,28 @@
   owner "root"
   mode "00755"
 end
+
+ruby_block "run_state_apache2_restart" do
+  block do
+    node.run_state['restart_apache2_needed'] = true
+  end
+  action :nothing
+end
+
+ruby_block "run_state_zabbix-server_restart" do
+  block do
+    node.run_state['restart_zabbix-server_needed'] = true
+  end
+  action :nothing
+end
+
+service "apache2" do
+  action :restart
+  only_if { node.run_state['restart_apache2_needed']  == true }
+end
+
+service "zabbix-server" do
```

Comment:
We already have a `service "zabbix-server"` resource defined [earlier](https://github.com/pu239ppy/chef-bach/blob/multiple_roles_for_disk_exclusion/cookbooks/bcpc/recipes/zabbix-head.rb#L160-L163) in the same recipe and I'd assume that this definition overwrites the previous one. We should have just one resource for enable/start/restart zabbix-server to keep things simple.